endpoints/endpoints.json

```diff
@@ -9,6 +9,16 @@
   },
   "GET /api/articles": {
     "description": "serves an array of all articles",
+    "queries": [
+      "title",
+      "topic",
+      "author",
+      "body",
+      "created_at",
+      "votes",
+      "asc",
```

Comment:
you could be more specific how these queries work, some of them work for order and some for sort_by